test/e2e/testnet/setup.go

```diff
@@ -15,8 +15,8 @@ import (
 func MakeConfig(node *Node, opts ...Option) (*config.Config, error) {
 	cfg := app.DefaultConsensusConfig()
 	cfg.TxIndex.Indexer = "kv"
-	cfg.Mempool.MaxTxsBytes = 1 * GiB
-	cfg.Mempool.MaxTxBytes = 8 * MiB
+	cfg.Consensus.TimeoutPropose = config.DefaultConsensusConfig().TimeoutPropose
+	cfg.Consensus.TimeoutCommit = config.DefaultConsensusConfig().TimeoutCommit
```

Comment:
[no change needed][for other reviewers] these lines override

```
TimeoutPropose: 10s => 3s 
TimeoutCommit 11s => 1s
```

it's kinda confusing that these lines aren't the defaults we use:
- https://github.com/celestiaorg/celestia-core/blob/6333e56ff954fae056f628ad0fe459409a0b3fcd/config/config.go#L989
- https://github.com/celestiaorg/celestia-core/blob/6333e56ff954fae056f628ad0fe459409a0b3fcd/config/config.go#L995

Reply:
In the e2e tests, we initially used the default consensus configurations with timeout commit and timeout propose set to 1 and 3 seconds, respectively. However, with the introduction of the benchmark test (this [PR](https://github.com/celestiaorg/celestia-app/pull/3612/files#diff-3d744f73561b631eaa5fc3b543a34858ab95ba21b53c768fdcffcb6b58f954bdR16)), we aligned the configurations with those used by celestia-app, which set higher values for timeout commit and propose. These higher values caused the normal e2e tests to time out. In this PR, I have reset the default timeouts to 1 and 3 seconds, and increased them only for the benchmark tests.

Reply:
PS: decreasing timeout commit and propose also make the e2e tests faster.

Reply:
ahh got it, thanks for clarifying!